libs/blocks/locui/langs/view.js

```diff
@@ -2,21 +2,23 @@ import { html } from '../../../deps/htm-preact.js';
 import { languages } from '../utils/state.js';
 import { rollout, showLangErrors, showUrls } from './index.js';
 
-function getPrettyStatus(status) {
+function getPrettyStatus({ status, queued }) {
```

Comment:
Maybe add a default value to avoid any "Cannot read property of undefined" error?

```suggestion
function getPrettyStatus({ status, queued } = {}) {
```